### PR TITLE
fix: add learn more link to the E2EI enrollment screen

### PR DESCRIPTION
**Pull request by typfel**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

Add missing "learn more" link to the E2EI enrollment screen

### Attachments

Simulator Screenshot - iPhone 15 Pro - 2024-03-07 at 00 33 25

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: wire-ios/Wire-iOS/Sources/UserInterface/Components/Views/MarkdownTextView.swift [hunk 1]
```diff
@@ -698,6 +698,23 @@ extension DownStyle {
         style.h3Size = style.h1Size
         return style
     }()
+
+    /// The style used during the login flow
+    static var login: DownStyle = {
```
Comment (agisilaos) [inline wire-ios/Wire-iOS/Sources/UserInterface/Components/Views/MarkdownTextView.swift:703]: Is this style used for the copy related to the link? If so we can rename it to:

```suggestion
    /// The style used during the login flow
    static var loginExternalLinkStyle: DownStyle = {
```

**Review by agisilaos — COMMENTED** [review]
Left a question and a suggestion

**Review by KaterinaWire — APPROVED** [review]

File: wire-ios/Wire-iOS/Sources/UserInterface/Components/Views/MarkdownTextView.swift (hunk 1 above)
Comment (typfel) [reply wire-ios/Wire-iOS/Sources/UserInterface/Components/Views/MarkdownTextView.swift:703]: It's not only related to links it the style used in the login flow. Paragraph styling and so on.

Comment (agisilaos) [reply wire-ios/Wire-iOS/Sources/UserInterface/Components/Views/MarkdownTextView.swift:703]: Then no need for change I guess. @typfel

**Review by agisilaos — APPROVED** [review]
Looks good! Thank you @typfel

**Pull request merged**